src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,202 @@
+    {children}
+  </div>
+)
+
+/* Collection Details: displays collection stats within Banner  */
+const CollectionDetails = ({
+  collection,
+  rank,
+  hovered,
+}: {
+  collection: TrendingCollection
+  rank: number
+  hovered: boolean
+}) => (
+  <Column className={styles.collectionDetails} paddingTop="16">
+    <div>
+      <span className={styles.volumeRank}>#{rank} volume in 24hr</span>
+    </div>
+    <Row>
+      <Box as="span" marginTop="16" className={clsx(header1, styles.collectionName)}>
+        {collection.name}
+      </Box>
+      {collection.isVerified && (
+        <Box as="span" marginTop="24">
+          <VerifiedIcon height="32" width="32" />
+        </Box>
+      )}
+    </Row>
+    <Row className={bodySmall} marginTop="12" color="explicitWhite">
+      <Box>
+        <Box as="span" color="darkGray" marginRight="4">
+          Floor:
+        </Box>
+        {collection.floor ? formatEthPrice(collection.floor.toString()) : '--'} ETH
+      </Box>
+      <Box>
+        {collection.floorChange ? (
+          <Box as="span" color={collection.floorChange > 0 ? 'green200' : 'error'} marginLeft="4">
+            {collection.floorChange > 0 && '+'}
+            {formatChange(collection.floorChange)}%
+          </Box>
+        ) : null}
+      </Box>
+      <Box marginLeft="24" color="explicitWhite">
+        <Box as="span" color="darkGray" marginRight="4">
+          Volume:
+        </Box>
+        {collection.volume ? putCommas(+toSignificant(collection.volume.toString())) : '--'} ETH
+      </Box>
+      <Box>
+        {collection.volumeChange ? (
+          <Box as="span" color={collection.volumeChange > 0 ? 'green200' : 'error'} marginLeft="4">
+            {collection.volumeChange > 0 && '+'}
+            {formatChange(collection.volumeChange)}%
+          </Box>
+        ) : null}
+      </Box>
+    </Row>
+    <Box
+      as="a"
+      className={clsx(buttonMedium, styles.exploreCollection)}
+      backgroundColor={hovered ? 'blue400' : 'grey700'}
+      href={`#/nfts/collection/${collection.address}`}
+      style={{ textDecoration: 'none' }}
+    >
+      Explore collection
+    </Box>
+  </Column>
+)
+
+/* Carousel Progress indicators */
+const CarouselProgress = ({
+  length,
+  currentIndex,
+  setCurrent,
+}: {
+  length: number
+  currentIndex: number
+  setCurrent: Dispatch<SetStateAction<number>>
+}) => (
+  <Center marginTop="16">
+    {Array(length)
+      .fill(null)
+      .map((value, carouselIndex) => (
+        <Box
+          cursor="pointer"
+          paddingTop="16"
+          paddingBottom="16"
+          onClick={(e) => {
+            e.preventDefault()
+            e.stopPropagation()
+            setCurrent(carouselIndex)
+          }}
+          key={carouselIndex}
+        >
+          <Box
+            as="span"
+            display="inline-block"
+            className={clsx(styles.carouselIndicator, carouselIndex === currentIndex && styles.carouselIndicatorActive)}
```

Comment:
nit: can be easier to read as `clsx(styles.carouselIndicator, { [styles.carouselIndicatorActive]: carouselIndex === currentIndex }`